_includes/tutorials/naming-changelog-repartition-topics/kstreams/markup/dev/run-consumer-names-with-filter.adoc

```diff
@@ -0,0 +1,54 @@
+////
+  This is a sample content file for how to include a console consumer to the tutorial, probably a good idea so the end user can watch the results
+  of the tutorial.  Change the text as needed.
+
+////
+
+Now stop the console-prodcer (if you haven't already) and start a console-consumer (for the last time I promise!)
+
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/naming-changelog-repartition-topics/kstreams/code/tutorial-steps/dev/console-consumer.sh %}</code></pre>
++++++
+
+You should see the following output:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/naming-changelog-repartition-topics/kstreams/code/tutorial-steps/dev/expected-output-names-with-filter.txt %}</code></pre>
++++++
+
+Looking at the last three records you see `2-4, 2-5, 2-6`.  This output is correct as you produced six records with the same key, so the final count is accurate.
+
+You have updated your topology and re-used the existing state stores and internal topics!
+
+
+
+
+. Always name sateful operators
+. If you haven't named your stateful operators and you need to update your toplogy, use the `https://docs.confluent.io/current/streams/developer-guide/app-reset-tool.html[Application Reset Tool]` to reprocess records (tutorial coming soon!).
+
```

Comment:
Does it make sense to put these two points under the conclusion header, too?

Reply:
yes will do